compact/far_backward/definitions.xml

```diff
@@ -298,5 +294,3 @@
     <constant name="LumiDirect_DZ"        value="0.2*m"/>
 
   </define>
-
-</lccdd>
```

Comment:
id.

Reply:
Hello why you would like to remove the line connected to Lumi direct photon detector from definitions.xml file?

Reply:
The LumiDirect_DZ constant is untouched, it is just the lccdd tag which is under discussion.

Reply:
Ok, thank you for explanation.

Reply:
Let's leave as is.